### ISSUE TITLE
i get Steam returned "General Failure" . When  adding   the  phone

### ISSUE THREAD

**Issue by kWAYTV** [Jessecar96/SteamDesktopAuthenticator#750]
i get Steam returned "General Failure" . When  adding  a phone number i guess some request or endpoint changed in  steam or  idk

**Comment by dyc3** [comment]
Nah, steam just does that sometimes. Add the phone number outside of SDA in steam itself.

**Comment by SuperAtom777** [comment]
> i get Steam returned "General Failure" . When adding a phone number i guess some request or endpoint changed in steam or idk

Change your account password, it worked for me

**Comment by AFoSZz** [comment]
I have the same issue, and I have the number added on Steam already. I even tried removing the number and re-adding it both using the SDA, and Steam itself, yet it still says the error..

**Comment by VoicelessRoar** [comment]
Same! The lazy dev needs to come and fix this shit

**Comment by VoicelessRoar** [comment]
@Jessecar96 
![image](https://user-images.githubusercontent.com/95093539/209037251-7098f221-b1c8-4aba-991c-4e99d0f8a732.png)
